cmd/flux/main_test.go

```diff
@@ -203,14 +206,32 @@ func (cmd *cmdTestCase) runTestCmd(t *testing.T) {
 
 	var expected string
 	if cmd.goldenValue != "" {
-		expected = cmd.goldenValue
+		if cmd.templateValues != nil {
+			tmpl := template.Must(template.New("golden").Parse(cmd.goldenValue))
```

Comment:
Can you please extract this to a func so we don't have to duplicate the template code.

Reply:
got it